packages/ui/src/components/Drawer/style.css

```diff
@@ -0,0 +1,4 @@
+.backdrop-drawer {
+  padding: 0px !important;
```

Comment:
This is to avoid as much as we can, `!important` like `z-index` can quickly become a war of this thing in the code base